src/sections/About.jsx

```diff
@@ -20,9 +25,85 @@ const AboutContainer = styled.div`
     // aspect-ratio: 428 / 1344;
   }
 `
+
+const AboutBg = styled.img`
+  position: absolute;
+  top: 0;
+  width: 100%;
+  height: auto;
+  // Makes background non-selectable
+  -moz-user-select: none;
+  -webkit-user-select: none;
+  user-select: none;
+`
+
+const AboutTitle = styled.h2`
+  font-family: 'HK Grotesk';
+  font-weight: bold;
+  text-align: center;
+  color: #4D4B4F;
+  z-index: 1;
+  width: 100vw;
+  display: block;
+  position: relative;
+  top: 15%;
+  font-size: 3em;
+  width: 80%;
+  text-align: center;
+`
+
+const AboutDescription = styled.p`
+  font-family: 'HK Grotesk';
+  top: 5px;
+  font-size: 1em;
+  color: #4D4B4F;
+  z-index: 1;
+  position: relative;
+  width: 50%;
+  left: 10%;
+  margin-top: 20%;
+`
+
+const swingSign = keyframes`
```

Comment:
The animation looks wrong for me (goes up and down). Looks as if the origin point of the transformation is center left. 

May need to tweak the `transform-origin` attribute in the CSS as well